src/index.ts

```diff
@@ -106,16 +103,14 @@ async function main() {
 			getFlashArbMessages,
 			botClients,
 			account,
-			botConfig.offerAssetInfo,
-			[botConfig.profitThreshold2Hop, botConfig.profitThreshold3Hop],
+			botConfig,
 		);
 	} else {
 		await sendSlackMessage("loop without mempool or skip not implemented yet", slackClient, botConfig.slackChannel);
 		return;
 	}
 	// main loop of the bot
 	await loop.fetchRequiredChainData();
-	loop.setGasFees([botConfig.txFee2Hop, botConfig.txFee3Hop]);
```

Comment:
Is gas fee setting now handled elsewhere?

Reply:
yes, the gas fees are already set in the config initialisation, dont need to set them manually anymore in a separate function